sources/ITAddressBookMgr.h

```diff
@@ -224,6 +224,10 @@
 #define KEY_SEND_NEW_OUTPUT_ALERT             @"Send New Output Alert"
 #define KEY_SEND_SESSION_ENDED_ALERT          @"Send Session Ended Alert"
 #define KEY_SEND_TERMINAL_GENERATED_ALERT     @"Send Terminal Generated Alerts"
```

Comment:
I think it makes more sense for this to be an advanced setting than a per-profile setting. You don't need to add a KEY_xxx constant for advanced settings.